extensions/amp-base-carousel/0.1/test-e2e/test-arrows.js

```diff
@@ -23,102 +23,85 @@ import {
   getSlide,
 } from './helpers';
 
-
-describes.endtoend('AMP carousel arrows', {
+/** The total number of slides in the carousel */
+const SLIDE_COUNT = 7;
+const pageWidth = 600;
+const pageHeight = 600;
+
+describes.endtoend('AMP carousel arrows when non-looping', {
+  testUrl: 'http://localhost:8000/test/manual/amp-base-carousel/non-looping.amp.html',
+  experiments: ['amp-base-carousel'],
```

Comment:
I don't think layers is fully launched yet, so I think we should still add the experiment here to toggle it on.

Reply:
What is layers needed for? I am confused why the tests pass without the experiment on

Reply:
Layers changes how the slides are detected for layout. While the carousel should work without it, it may take a bit longer for some slides to be laid out. I do want to make sure that the layers code keeps working with carousel and I don't do something to break it.

Reply:
Got it, I'll re-add the experiment